pkg/chunk/aws_storage_client.go

```diff
@@ -76,6 +76,12 @@ var (
 		// metric names.
 		Buckets: prometheus.ExponentialBuckets(1, 4, 6),
 	})
+	dynamoQueryRetryCount = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "cortex",
+		Name:      "dynamo_query_retry_count",
+		Help:      "Number of retries per DynamoDB operation.",
+		Buckets:   []float64{1, 2, 3, 5, 10, 15, 20},
```

Comment:
I think you should add a bucket for 0. As is, "no retries" shares a bucket with "one retry", and if we want to find out how many have no retries we'll only ever get an interpolation.